simd.dd

```diff
@@ -57,9 +57,25 @@ import core.simd;
 ---
 
 	$(P These types and operations will be the ones defined for the architecture
-	the compiler is targetting. If a particular CPU family has varying
+	the compiler is targeting. If a particular CPU family has varying
 	support for vector types, an additional runtime check may be necessary.
-	The compiler does not emit runtime checks; those must be done by the programmer.
+	The compiler does not emit runtime checks; those must be done by the 
+	programmer.
+	)
+	
+	$(P It is possible to test if the architecture the compiler is targeting
+	supports SIMD types by using:)
+---
+version(D_SIMD)
```

Comment:
I removed the version(D_SIMD) text. Sorry if you've mentioned something "time and again". I only first started using SIMD types like 3 days ago and was not aware of this. Some thing you may find obvious in the documentation are not as obvious to people with less experience. I've been trying to add more details to areas where I had some difficulty (I probably spent a few hours trying to figure out why it wouldn't compile when I didn't use -m64).